packages/docs/src/routes/community/groups/index.mdx

```diff
@@ -62,5 +59,4 @@ and spread your love for innovation!
 
 #### United Kingdom
 
-- <MeetupLogo />
-  <GroupLink link="https://www.meetup.com/qwik-london/">Qwik London Meetup Group</GroupLink>
+- <MeetupLogo /> <GroupLink link="https://www.meetup.com/qwik-london/">Qwik London Meetup Group</GroupLink>
```

Comment:
This doesn't fix anything, I just inlined these two lines because all other bullets above are inlined.